app/views/jobs/index.atom.builder

```diff
@@ -1,11 +1,13 @@
-atom_feed :language => 'en-US' do |feed|
-  feed.title("IndyHackers Job Board")
+atom_feed language: 'en-US' do |feed|
+  feed.title('IndyHackers Job Board')
   feed.updated(@jobs.first.created_at)
 
   @jobs.each do |job|
     feed.entry(job) do |entry|
-      entry.title(job.title)
-      entry.content(MARKDOWN.render(job.description).html_safe, :type => 'html')
+      title = job.title
+      title += ' (' + job.salary + ')' unless job.salary.blank?
```

Comment:
Fully dig this... but I am curious about two things:

1) A ternary operator to assign the title value I think could read a tad easier... but that is probably 100% personal preference
2) The choice to use concatenation as opposed to interpolation

It has been a hella long time since I reviewed any code, so thanks for putting up with me :-)

Reply:
Yeah, this could be cleaner. It's been over a year since I've really used ruby, so I'm out of practice. Lemme give that a shot.

Reply:
Changed it up. Made it better. Thanks for calling it out!